src/option.c

```diff
@@ -7470,7 +7470,7 @@ set_context_in_set_cmd(
 		while (s > xp->xp_pattern && *(s - 1) == '\\')
 		    --s;
 		if ((*p == ' ' && (xp->xp_backslash == XP_BS_THREE && (p - s) < 3))
-			|| (*p == ',' && (flags & P_COMMA) && ((p - s) % 1) == 0)
+			|| (*p == ',' && (flags & P_COMMA))
```

Comment:
what if `(p -s )` is negative?

Reply:
👀 is that allowed?

Reply:
Is it still not zero?

Reply:
I'm not sure. (but probably)

Reply:
I pointed out what the logic is doing elsewhere already but `p-s` cannot be negative here.